src/estimagic/visualization/estimation_table.py

```diff
@@ -265,184 +358,715 @@ def tabular_tex(
         )
     latex_str = latex_str.split("\\bottomrule")[0]
     if show_footer:
-        stats_str = footer_df.to_latex(**default_options)
-        stats_str = (
-            "\\midrule" + stats_str.split("\\midrule")[1].split("\\bottomrule")[0]
-        )
+        if "Observations" in stats.index.get_level_values(0):
+            stats = stats.copy(deep=True)
+            stats.loc[("Observations",)] = _add_multicolumn_left_format(
```

Comment:
It would make sense to apply this to all other integer statistics (e.g. if the "Observations" index entry is called differenctly. Alternatively, I would let the user enter a list of statistics which should be treated as integers (and hence left aligned)